syntax = "proto3";

import "packages/hal/proto/device.proto";
import "packages/core/proto/timestamp.proto";

package hal;

message JoystickSample {
    Device device = 1;
    core.SystemTimestamp systemTimestamp = 2;
    core.HardwareTimestamp hardwareTimestamp = 3;
    repeated float axis = 4;
}
